components/view_image/view_image.jsx

```diff
@@ -203,6 +205,33 @@ export default class ViewImageModal extends React.PureComponent {
         this.setState({showCloseBtn: false});
     }
 
+    handleZoomIn = () => {
+        let newScale = this.state.scale;
+        newScale = Math.min(newScale + ZoomSettings.SCALE_DELTA, ZoomSettings.MAX_SCALE);
+        this.setState(() => ({
```

Comment:
Can this be simplify this to be ` this.setState({scale: newScale);` since we are not using `this.setState((prevState) => ({` prevState?

Reply:
Changed.